includes/md_logging.hpp

```diff
@@ -2,17 +2,26 @@
 
 #include <iostream>
 #include <map>
+#include "logging_sink.grpc.pb.h"
 
 namespace mdlog {
 
-std::string escape_char(const std::string &str, const char &c);
+    class MdLog {
+    public:
+        explicit MdLog(const std::string &token_, std::shared_ptr<logging_sink::LoggingSink::Stub>& _client);
+        void log(const std::string &level, const std::string &message);
+
+        void info(std::string message);
 
-void log(const std::string level, const std::string message);
+        void warning(std::string message);
```

Comment:
yeah, please keep reminding me. Is there a stylecheck I can install that will do that?

Reply:
So I went on looking for clang-tidy... and found this https://stackoverflow.com/a/61148162/667433 :-)
I think it would be great to enable clang-tidy on this repo

Reply:
Agreed about clang-tidy!
Passing by const-ref=good, passing by value=good (you can move out of it), passing by const value = bad (can't move out of it), passing by mutable lvalue reference = very bad